infra/tf/README.md

```diff
@@ -10,7 +10,7 @@ Terraform is our tool of choice for automating our 'cloud infrastructure'. In pa
 * [Going public](#going-public)
 * [Publishing a release](#publishing-a-release)
 * [Updating a module for a new version of Terraform](#updating-a-module-for-a-new-version-of-Terraform)
-* [How to layout/structure a Terraform Project](How-to-layout/structure-a-Terraform-Project)
+* [How to layout/structure a Terraform Project](#how-to-layoutstructure-a-terraform-project)
```

Comment:
Thanks for the fix! 
I did a little research and this can be either `#how-to-layoutstructure-a-terraform-project` or `#how-to-layout/structure-a-Terraform-project`
But the most important thing is not to forget the hashtag! 
Thanks for teaching me something today :)